### DISCUSSION TITLE
CLI for Windows?

### DISCUSSION THREAD

**Discussion by essteewhy** [upscayl/upscayl#822] [Q&A]
Hello, just curious if there's a CLI version for Windows?  I didn't see anything about it for windows in the documentation..  thanks!

**Comment by aaronliu0130** [comment]
> - **Is there a CLI available?**
>   - The CLI tool is called [upscayl-ncnn](https://github.com/upscayl/upscayl-ncnn).

![compulsory rickrolls are issued for duplicates of issues and the FAQ](https://user-images.githubusercontent.com/11874211/218282922-efab6795-84af-4cfb-9400-c28c8729ea78.gif)

read the faq lol
